src/sentry/models/identity.py

```diff
@@ -65,19 +67,3 @@ class Meta:
         app_label = 'sentry'
         db_table = 'sentry_identity'
         unique_together = (('idp', 'external_id'),)
-
-
-class UserIdentity(Model):
```

Comment:
Let's not actually drop this yet until a later migration.

Reply:
[Addressed in this comment](https://github.com/getsentry/sentry/pull/6708#issuecomment-354855601)